doc/api/n-api.md

```diff
@@ -1737,11 +1737,10 @@ napi_status napi_get_value_string_utf8(napi_env env,
 - `[in] buf`: Buffer to write the UTF8-encoded string into. If NULL is passed
  in, the length of the string (in bytes) is returned.
 - `[in] bufsize`: Size of the destination buffer.
-- `[out] result`: Number of bytes copied into the buffer including the null.
-terminator. If the buffer size is insufficient, the string will be truncated
-including a null terminator.
+- `[out] result`: Number of bytes copied into the buffer, excluding the null
+terminator.
```

Comment:
I think it would be better to move that sentence to be part of the `[in] bufsize` parameter comment.  That's where it's relevant to call out that the string will be truncated, but still include a null terminator.